### ISSUE TITLE
add support for Chocolately

### ISSUE THREAD

**Issue by cilerler** [roboll/helmfile#894]
[Chocolately](https://chocolatey.org/) is the standard package manager for Windows; please add support for it

**Comment by mumoshu** [comment]
Hey!

Have you ever tried using Helmfile on windows?

I'm willing to do so myself but don't have a Windows machine for developing and even a test suite for running Helmfile on Windows.
I'm awaiting someone to contribute here :)

**Comment by cilerler** [comment]
I tried, but no luck because of the PS1 files.  I can do the test work. However, you need to start replacing sh files with ps1 files.

**Comment by mumoshu** [comment]
@cilerler Thanks! Are you talking about shells scripts under https://github.com/roboll/helmfile/tree/master/test/integration specifically, or anything else?

**Comment by cilerler** [comment]
Yes, since Shell can not be run on Windows, but Powershell can work on both platforms, those should be written PS1.  And then we may start to create a package for chocolate, I guess

**Comment by mumoshu** [comment]
@cilerler Thanks for clarifying!

I'm fine with co-maintaining PS1 version of the integration test script. (I'm pretty new to powershell and doubt I have time to fully migrate all the scripts to PS1 but co-maintaining should be feasible
